package.json

```diff
@@ -69,15 +73,20 @@
     "css-loader": "^1.0.1",
     "cssnano": "^4.1.7",
     "downshift": "^3.1.5",
+    "dotenv": "^6.2.0",
+    "dotenv-webpack": "^1.5.7",
+    "expect": "^23.6.0",
```

Comment:
is this necessary? I believe expect comes with Jest, except you're using it for something other than test assertion?

Reply:
Feedback duly noted. I am on it now.

Reply:
Yes. You are right. Done